Library/Homebrew/cask/utils.rb

```diff
@@ -79,6 +79,18 @@ def self.path_occupied?(path)
       path.exist? || path.symlink?
     end
 
+    sig { params(name: String).returns(String) }
+    def self.token_from(name)
+      name.downcase
```

Comment:
Would be good to have even a simple test case for this.

Reply:
I added one to `create_spec.rb` for now.